llvm/test/Verifier/RemoveDI/llvm.dbg.intrinsic-dbg-attachment.ll

```diff
@@ -0,0 +1,55 @@
+; RUN: llvm-as -disable-output <%s 2>&1 | FileCheck %s
+define void @foo() {
+entry:
+    #dbg_value(
+      ptr undef,
+      !DILocalVariable(scope: !1),
+      !DIExpression(),
+      !{})
+; CHECK-LABEL: invalid #dbg record location
```

Comment:
what's invalid about the "location" and which of the 3 locations (position, src-loc, var-loc) is this referring to?

Reply:
The location overload strikes again... I'll update the verifier check to state more clearly that it's a DILocation.

Reply:
SGTM thanks